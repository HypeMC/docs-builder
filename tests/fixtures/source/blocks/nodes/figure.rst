.. figure:: images/logo.png
   :alt: Symfony Logo
   :width: 200px

I am a paragraph AFTER the figure. I should not be included as the
caption for the above figure.

.. figure:: images/logo.png

    But I am a caption *for* the figure above.
