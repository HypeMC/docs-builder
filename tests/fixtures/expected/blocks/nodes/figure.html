<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>

<figure>
    <!-- REMOVE: src is empty just because the test doesn't utilize the full stack, with CopyImagesListener -->
    <img src="" alt="Symfony Logo" width="200px" />
</figure>
<p>I am a paragraph AFTER the figure. I should not be included as the
caption for the above figure.</p>

<figure>
    <img src="" />

    <figcaption><p>But I am a caption <em>for</em> the figure above.</p></figcaption>
</figure>

</body>
</html>
